### PR TITLE
change the langauge around kinds based on feedback

### PR DESCRIPTION
**Pull request by stew**

### CODE REVIEW

**Comment by non** [comment]
Other than those two little comments, look good to me :+1:

File: docs/src/main/tut/functor.md [hunk 1]
```diff
@@ -54,6 +55,20 @@ implicit def function1Functor[In]: Functor[Function1[In, ?]] =
   }
 ```
 
+Also of not in the above example, is that we created a functor for
```
Comment (ceedubs) [inline docs/src/main/tut/functor.md]: s/of not/of note/

File: docs/src/main/tut/functor.md [hunk 2]
```diff
@@ -54,6 +55,20 @@ implicit def function1Functor[In]: Functor[Function1[In, ?]] =
...
+Function1, which is a type which normally has two type holes. We
+however constrained one of the holes to be the `In` type, leaving just
+one hole for the return type. In this above example, we are
+demonstrating the use of the
+[kind-projector compiler plugin](https://github.com/non/kind-projector),
+This compile plugin lets us more easily change the number of type
```
Comment (ceedubs) [inline docs/src/main/tut/functor.md]: s/compile/compiler/

**Comment by non** [comment]
:+1:

**Pull request merged**